editor/file_system/dependency_editor.cpp

```diff
@@ -405,27 +405,58 @@ void DependencyEditorOwners::show(const String &p_path) {
 	_fill_owners(EditorFileSystem::get_singleton()->get_filesystem());
 	popup_centered_ratio(0.3);
```

Comment:
Moving the popup after all updates fixes dialog going off screen, but not sure what to do about the size.
As a last resort you can use minimum height + `reset_size()`.